tools/src/versioning/android/versionCxx/index.ts

```diff
@@ -153,6 +153,10 @@ async function versionJavaLoadersAsync(
           find: new RegExp(`\\b(System\\.loadLibrary\\("expo.*)("\\);?)`, 'g'),
           replaceWith: `$1_${abiName}$2`,
         },
+        {
+          find: new RegExp(`\\b(SoLoader\\.loadLibrary\\("expo.*)("\\);?)`, 'g'),
```

Comment:
Two questions:
- Why not to use `/\b(SoLoader\.loadLibrary\("expo.*)("\);?)/g` syntax?
- Why not to join this rule with the previous one `System.loadLibrary`?

Reply:
Another note that `("expo.*)` could potentially expand to something longer and unexpected 🤔 Maybe `[^"]` instead of `.`?

Reply:
that's my fault during refactored the code for versioning expo-av. i've updated accordingly. thanks!